chore: require minimal reproducible example for bugs (#517)
uuidjs/uuid: .github/ISSUE_TEMPLATE/bug_report.md

## How to reproduce

> [Provide a [Minimal, Complete, and Verifiable](https://stackoverflow.com/help/minimal-reproducible-example) so that we (the `uuid` maintainers) can reproduce the problem. Bug reports without minimal reproducible example may be closed without further investigations.]

Reviewer: I would be more blunt about this ...

```suggestion
**REQUIRED**: [Provide a [Minimal, Complete, and Verifiable](https://stackoverflow.com/help/minimal-reproducible-example) so that we (the `uuid` maintainers) can reproduce the problem.]  Issues that do not provide this will be closed without investigation.
```
